cosmos.gemspec

```diff
@@ -87,6 +87,7 @@ spec = Gem::Specification.new do |s|
   s.add_development_dependency 'reek', '~> 1'
   s.add_development_dependency 'roodi', '~> 4'
   s.add_development_dependency 'guard', '~> 2'
+  s.add_development_dependency 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
You can do something like this in the main Gemfile.    What is requiring wdm that is not installing it?